libs/langchain-community/src/document_loaders/web/cheerio.ts

```diff
@@ -78,7 +85,9 @@ export class CheerioWebBaseLoader
     caller: AsyncCaller,
     timeout: number | undefined,
     textDecoder?: TextDecoder,
-    options?: CheerioOptions
+    options?: CheerioOptions & {
+      headers: Headers;
```

Comment:
Headers shouldn't be required if options are passed
```suggestion
    options?: CheerioOptions & {
      headers?: Headers;
```